MoneyTransferClient/RunWorkflow.cs

```diff
@@ -0,0 +1,38 @@
+// @@@SNIPSTART money-transfer-project-template-dotnet-start-workflow
+using Temporalio.MoneyTransferProject.Workflow;
+using Temporalio.MoneyTransferProject.Shared;
+using Temporalio.Client;
+
+// Connect to the Temporal server
+var client = await TemporalClient.ConnectAsync(new("localhost:7233"));

```

Comment:
```suggestion
var client = await TemporalClient.ConnectAsync(new("localhost:7233") { Namespace = "default" });
```

Don't have to do this, but I see we do it in the Python one because for most production users, they won't use the default namespace